### DISCUSSION TITLE
General discussions

### DISCUSSION THREAD

**Discussion by deborahtrez** [curiouslearning/CuriousReaderInteractiveFunctionsH5P#227] [#OSEQ Hackathon]
I would like to share my thoughts on ....

**Comment by MarcDemo** [comment]
Hi, i was localizing a book called hide and seek but it lacks images on page 2,3 and 4 so should i just continue or the images will be provided.

**Reply by deborahtrez** [reply]
Hi Mark,

I know I already replied to you through our email interaction but the solution might help other participants too so I thought it would be good to share.

About the missing images, that could be a slight error.  
They must have gotten deleted by mistake, which is why we are giving users read access so that they can edit on their own copies. 
However, regarding the graphics,  please use the graphics that have been provided to you in [this folder](https://drive.google.com/drive/folders/1zQ8YMsCG24n6gpsC45h3DKZIudc47MR9).
Just navigate to the folder of the book that you're working on and you will find the images in here. 😊

Thank you Mark.
